src/portals/hope/features/member/tabs/payments-tab/PaymentsTab.tsx

```diff
@@ -179,6 +182,33 @@ const MemberTransactionsTable: React.FC<{ memberId: string }> = ({
               <PaymentColumn title={transaction.status?.toString()}>
                 {transaction.status}
               </PaymentColumn>
+              <PaymentColumn title="Cancel Auto Rescue">
+                <Button
+                  onClick={(e) => {
+                    e.preventDefault()
+                    toast.promise(
+                      cancelAutoRescueMutation({
+                        variables: {
+                          transactionId: transaction.id!,
+                        },
+                      }),
+                      {
+                        loading: 'Cancelling auto rescue...',
+                        success: ({ data: response }) => {
+                          if (response?.cancelAutoRescue === null) {
+                            return null
+                          }
+
+                          return 'Auto rescue successfully canceled!'
+                        },
+                        error: 'Could not cancel auto rescue',
+                      },
```

Comment:
If you return the mutated resource as mentioned in my other comment, you can simplify this one to

```suggestion
                      {
                        loading: 'Cancelling auto rescue',
                        success: 'Auto rescue successfully canceled',
                        error: 'Could not cancel auto rescue',
                      },
```

Also curious, which field is changed on a `Transaction` when the mutation succeeds? Because if the new resource is returned, the cache will be updated automatically 😀

Reply:
The status will be changed, but not when the mutation succeeds, because it starts an async flow that will eventually change the status when Adyen notifies back that the rescue cancellation succeeded. 😀